interfaces/cython/cantera/kinetics.pyx

```diff
@@ -365,6 +365,26 @@ cdef class Kinetics(_SolutionBase):
         def __get__(self):
             return get_reaction_array(self, kin_getDeltaSSEntropy)
 
+    property heat_release_rate:
+        """
+        Get the total volumetric heat release rate [W/m^3].
+        """
+        def __get__(self):
+            return - np.sum(self.partial_molar_enthalpies *
+                            self.net_production_rates, 0)
+
+    property heat_production_rates:
+        """
+        Get the volumetric heat production rates [W/m^3] on a per-reaction
+        basis. The sum over all reactions results in the total volumetric heat
+        release rate.
+        Example: C. K. Law: Combustion Physics (2006), Fig. 7.8.6
+
+        >>> gas.heat_production_rates[2]  # heat production rate of the 2nd reaction
```

Comment:
I believe this should be the third reaction, or else the index should be 1

Reply:
Good catch - you are correct! I had just cut&paste what was there already and had not noticed this issue.

Reply:
fixed.